docs/reference/TensorFlow.md

```diff
@@ -1,405 +1,394 @@
-
-With the launch of TensorFlow Lite for Microcontrollers, developers can run machine learning inference on extremely low-powered devices, like the Cortex-M microcontroller series. Watch the video to learn more about the announcement:
-
-[![An introduction to TensorFlow Lite](http://img.youtube.com/vi/DKosV_-4pdQ/0.jpg)](http://www.youtube.com/watch?v=DKosV_-4pdQ)
-
-# Getting Started
+<!--this needs to move-->
+**Important:** We recommend running the following commands from inside the Mbed CLI terminal that gets launched with the Mbed CLI Application. This is because it will be much quicker to set up, because it resolves all your environment dependencies automatically.
 
-**Before you begin**
+### Prerequisites
 
 Here is what you will need to complete the guide:
 
-* Computer that supports Mbed CLI (version 1.10.4)
-
-* NXP FRDM K66F
-
-* Mini-USB cable
-
-* Python 2.7. Using pyenv is recommended to manage python versions.
+* Ubuntu 20.04 LTS. If you are using a Windows machine, install [VirtualBox](https://www.virtualbox.org/wiki/Downloads)<!--originally you said 6.0 - that version's support ended in July. Can they install the latest version, or do they need 6.0 specifically?--> and then [Install Ubunto on it](https://www.virtualbox.org/wiki/Linux_Downloads).
 
-For Windows users, install Ubuntu 20.04 LTS in a VirtualBox. Refer to the following videos to set up:
+* [Mbed CLI (version 1.10.4)](../build-tools/install-and-set-up.html).
 
-**How to install VirtalBox 6.0.10 on Windows 10**
+* The [GNU Arm Embedded](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads) toolchain [that Mbed CLI supports](../build-tools/index.html).<!--Mbed OS supports ARMC 6.14, and many users rely on it. Can they use it?--> For Windows users, if you have already set up the Linux virtual environment, install the toolchain there.
 
-![IMAGE ALT TEXT HERE](http://img.youtube.com/vi/8mns5yqMfZk/0.jpg)
+    Configure Mbed CLI to find your GNU installation:
 
-[Watch here](http://www.youtube.com/watch?v=8mns5yqMfZk)
+    ```    
+    mbed config -G GCC_ARM_PATH <path_to_your_arm_toolchain>/bin
+    ```
 
-**How to install Ubuntu 20.04 on VirtualBox in Windows 10**
+* Make 3.82 or newer (if you are using an older version, see the troubleshooting section for instructions).
 
-![IMAGE ALT TEXT HERE](http://img.youtube.com/vi/x5MhydijWmc/0.jpg)
+    <span class="notes">**Note**: On a Mac, you might have to use gmake instead of make.</span>
 
-[Watch here](http://www.youtube.com/watch?v=x5MhydijWmc)
+* Python 2.7. We recommend [using pyenv to manage Python versions](https://pypi.org/project/pyenv/).<!--The install page for Mbed CLI now requires 3.7. Why are we asking for 2.7?-->
 
+* Git.
 
-**Getting started**
+* xxd.<!--unless is exists by default on all Linux computers?-->
```

Comment:
Well, you are right, it does come with Ubuntu although it could be the case some minimalist distros don't include it for some reason. But since the tutorial focuses on Ubuntu we could leave this out.

Reply:
```suggestion

```